mccn/wcs_importer.py

```diff
@@ -0,0 +1,70 @@
+from abc import ABC, abstractmethod
+from typing import Any, List
```

Comment:
Probably unimportant, but you may want to use `Sequence` over `List` due to invariance.

Reply:
I have just read that the typing library versions of generics have been deprecated. I will rethink this.

Reply:
Oh it's collections now. Instead of `from typing import Sequence`, you will now do `from collections.abc import Sequence`. It is to avoid the problem we saw ytd - `Mapping` vs `dict`. The only downside I can think of is if you use a type validation library like `pydantic` (which is used by `FastAPI` under the hood), the performance of `Sequence` is worse than `list` if your argument is a list.